docs/runbooks/VirtualMachineInstanceHasEphemeralHotplugVolume.md

```diff
@@ -0,0 +1,59 @@
+# VirtualMachineInstanceHasEphemeralHotplugVolume
+
+## Meaning
+
+The `VirtualMachineInstanceHasEphemeralHotplugVolume` alert is triggered when a 
+Virtual Machine Instance (VMI) contains an Ephemeral Hotplug Volume, which is defined
+as a hotplug volume that only exists in the VMI and will not persist during VM restart
+
+## Impact
+
+The `HotplugVolumes` Feature Gate will be deprecated in a future release and will
+be replaced by the `DeclarativeHotplugVolumes` Feature Gate. The two are mutually
+exclusive, and when `DeclarativeHotplugVolumes` is enabled, any remaining 
+ephemeral hotplug volumes will automatically be unplugged from any VMIs.
+
+If this alert is triggered, it is to inform of this future deprecation and to
+suggest steps to convert ephermeral volumes to persist ones.
+
+## Diagnosis
+
+To diagnose the cause of this alert, the following steps can be taken:
+
+1. Find the affected VM(s) and volume(s) that are reported in the Alert.
```

Comment:
> Find the affected VM(s)

@Dsanatar may want to how to list all vmis with matching label selector that was added here https://github.com/kubevirt/kubevirt/pull/15815